### PR TITLE
Check in training notebook

### PR DESCRIPTION
**Pull request by Katsutoshii**
Check in training notebook, remove print statements.

### CODE REVIEW

File: usl_models/train_flood_model.ipynb [hunk 1]
```diff
@@ -0,0 +1,151 @@
+{
```
Comment (waltaskew) [inline usl_models/train_flood_model.ipynb:1]: let's stick this someplace like a directory named 'examples' rather than at the package root

**Review by waltaskew — COMMENTED** [review]
let's stick this someplace like a directory named 'examples' rather than at the package root

**Comment by Katsutoshii** [comment]
> let's stick this someplace like a directory named 'examples' rather than at the package root

Done, moved to `usl_models/notebooks` folder.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (waltaskew) [inline .vscode/settings.json:1]: looks like this was accidentally included

**Review by waltaskew — APPROVED** [review]

**Pull request merged**